src/main/resources/the_warlordResources/localization/eng/WarlordMod-Power-Strings.json

```diff
@@ -189,5 +189,11 @@
     "DESCRIPTIONS": [
       "Gain #b%1$s #yTension at the start of each turn."
     ]
+  },
+  "the_warlord:ImprovisePower": {
+    "NAME": "Improvise",
+    "DESCRIPTIONS": [
+      "At the start of each turn, if you have no #yPosture, the next card you play is played #b%1$s times."
```

Comment:
This should be "... the first #b%1$s cards you play this turn are played an twice."